packages/electrode-bolt/config/babel/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Yup. We sorta have an issue open to track this, but it needs better documentation.

Quick explanation, some tools that include Babel support (`isparta-loader` is one example, but I think there are more) don't have an option to send it a config – they just check for `.babelrc` in a typical dir structure.